page-objects/navigationPage.ts

```diff
@@ -1,45 +1,60 @@
 import { Locator, Page } from "@playwright/test";
-export class NavigationPage {
-    private readonly page: Page;
-    readonly fromLayoutsMenuItem: Locator
-    readonly datePickerMenuItem: Locator
-    readonly smartTableMenuItem: Locator
-    readonly toastrMenuIteam: Locator
-    readonly tooltipMenuItem: Locator
+import { HelperBase } from "./helperBase";
+
+export class NavigationPage extends HelperBase {
+    readonly page: Page;
+
+    // Define locators
+    readonly fromLayoutsMenuItem: Locator;
```

Comment:
Let's using those object as the private object instead of public like this - https://www.shecodes.io/athena/5201-what-is-public-and-private-in-typescript#:~:text=Private%3A%20Members%20declared%20as%20private,name%20%3D%20name%3B%20this.